src/unix/os390-syscalls.c

```diff
@@ -136,6 +136,11 @@ static void maybe_resize(uv__os390_epoll* lst, unsigned int len) {
 }
 
 
+void uv__os390_cleanup(void) {
+  msgctl(uv_backend_fd(uv_default_loop()), IPC_RMID, NULL);
```

Comment:
I don't think this is sound, it may initialize the default loop while libuv is already partially torn down.

I'll merge this PR anyway because it only affects os390 but you might want to follow up on it.